docs/contributing.rst

```diff
@@ -69,6 +69,45 @@ found in the `virtualenv docs`_.
 
 .. _`virtualenv docs`: https://virtualenv.pypa.io
 
+Running certbot against a locally deployed Pebble CA
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+By following the instructions above, you can execute Certbot against a local instance
+of Pebble CA. This is useful to verify that the modifications done to the code makes
```

Comment:
```suggestion
of the Pebble CA. This is useful to verify that the modifications done to the code makes
```
I also personally like "Pebble ACME server" over "Pebble CA" but either is fine.

Also, optional nit: I think it'd be useful to make the first reference to "Pebble" in each of these sections link to https://github.com/letsencrypt/pebble. If you're a new dev, what's a Pebble?

Reply:
Did the modification, and put a link ref for Pebble.